sample-apps/SampleAppSpec.md

```diff
@@ -134,15 +134,17 @@ Asynchronous Gauge
 
 **Common Attributes for Random based metrics**
 
-These are Key Value pairs to be added on metrics.
+These are Key Value pairs to be added on metrics and will be tested for.
 ```
 {
 “signal”:     (string)“metric”
-“language”:   (string)<name of language used>
+“language”:   (string)<name of language used. Should be set to the name of the sample app preceeding "-sample-app" for standardization purposes> 
 “metricType”: (string)“random”
 }
 ```
 
+Upon implementation, the Metric names should also have the instance ID appended onto the end of the name like so "\<metric name\>_\<instance ID\>".  The instance ID can be retrieved through the INSTANCE_ID environment variable.  An example of what this would look like: `cpuUsage_a1b2c3d4e5f6g7h8`.
```

Comment:
Is there a better place to reflect metric naming requirements? I believe this could be worded a bit differently to represent the case where `INSTANCE_ID` does not exist.  
```suggestion
If the `INSTANCE_ID` environment variable exists then the sample app MUST append it to all metric names following an underscore(`_`). Example: `cpuUsage_a1b2c3d4e5f6g7h8`.
```

Reply:
I think this place makes sense to reflect metric naming requirements.  This wording also works in showing the what is required so I will be including this change.